LoopKit/QuantityFormatter.swift

```diff
@@ -195,19 +195,23 @@ public extension HKUnit {
     }
 
     var pickerFractionDigits: Int {
+        maxFractionDigits
+    }
+    
+    var maxFractionDigits: Int {
```

Comment:
RE: aliasing `pickerFractionDigits` to `maxFractionDigits`.  Since `maxFractionDigits` is the primary usage (used in many formatters throughout Loop), perhaps we just eliminate `pickerFractionDigits` and have the pickers refer to `maxFractionDigits`? One less thing code readers have to resolve (the question of what's the difference) to understand the code.